recipes/chakra-ui/index.ts

```diff
@@ -25,6 +25,110 @@ function wrapComponentWithChakraProvider(program: Collection<j.Program>) {
   return program
 }
 
+function updateLabeledTextFieldWithInputComponent(program: Collection<j.Program>) {
+  program
+    .find(j.TSInterfaceDeclaration)
+    .find(j.TSExpressionWithTypeArguments)
+    .forEach((path: j.ASTPath<j.TSExpressionWithTypeArguments>) => {
+      path.replace(
+        j.tsExpressionWithTypeArguments(
+          j.identifier("ComponentPropsWithoutRef"),
+          j.tsTypeParameterInstantiation([j.tsTypeQuery(j.identifier("Input"))]),
+        ),
+      )
+    })
+
+  return program
+}
+
+function replaceOuterDivWithFormControl(program: Collection<j.Program>) {
+  program
+    .find(j.JSXElement)
+    .filter((path) => {
```

Comment:
This pattern within `filter` functions is somewhat repeated in this file, and I opted for clarity over abstraction. Happy to change if the code-style says otherwise.

Open to stylistic and structural feedback.